### PR TITLE
SDPA-406 : Cannot add role to AD user

### PR DESCRIPTION
**Pull request by naomiaro**

### CODE REVIEW

**Comment by naomiaro** [comment]
Together with https://github.com/superdesk/superdesk-client-core/pull/343

File: superdesk/users/users.py [hunk 1]
```diff
@@ -45,22 +44,20 @@ def __init__(self, endpoint_name, app, service, endpoint_schema=None):
...
-                'type': 'string',
-                'readonly': self.readonly
+                'type': 'string'
             },
             'email': {
                 'unique': True,
                 'type': 'email',
-                'required': True
+                'required': True,
+                'readonly': self.readonly
```
Comment (mdhaman) [inline superdesk/users/users.py]: You could create users in LDAP (for testing) that might not have email. Maybe allow the email to editable.

Comment (naomiaro) [inline superdesk/users/users.py]: For testing if it's LDAP, a valid user should be created anyways really. Email seems to be the most important part.

Comment (petrjasek) [inline superdesk/users/users.py]: can it be both `readonly` and `required` at the same time?

Comment (naomiaro) [inline superdesk/users/users.py]: hmm yeah this might be a problem, it was ok for me but I should test bringing in a new user via LDAP

Comment (naomiaro) [inline superdesk/users/users.py]: Might be better even to change sometime to a different schema depending on if it's POST/PATCH

Comment (mdhaman) [inline superdesk/users/users.py]: Making `phone` field as editable is fine. But as per the video attached in the ticket, I cannot understand why the user does not have the `default` role assigned to the user. Does PA setup have `default` role.

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**